src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1983,10 +2083,30 @@ instance Semigroup LimitClause where
 
 instance Monoid LimitClause where
   mempty = Limit mzero mzero
-  mappend = (<>)
 
 -- | A locking clause.
-type LockingClause = Monoid.Last LockingKind
```

Comment:
Hmm, changing this type is also probably a breaking change.

Reply:
This is only used internally to wrap `LockingKind` which is the external one - I could alternatively wrap this one instead?

Reply:
Indeed! It's fine 😄